action_server/src/sema4ai/action_server/_rcc.py

```diff
@@ -356,12 +360,16 @@ def clean_tools_caches(self):
 def initialize_rcc(rcc_location: Path, robocorp_home: Optional[Path]) -> Iterator[Rcc]:
     global _rcc
 
-    rcc = Rcc(rcc_location, robocorp_home)
-    _rcc = rcc
-    try:
-        yield rcc
-    finally:
-        _rcc = None
+    if _rcc:
+        yield _rcc
```

Comment:
Minor: should have a return after this yield so that it doesn't even go to the other check.